### ISSUE TITLE
IBTrACS: Tropical cyclone best track data

### ISSUE THREAD

**Issue by fBedecarrats** [mapme-initiative/mapme.biodiversity#161]
From the dataset website: 
- Consolidates global tropical cyclone data at a central location.
- Available in a single file or subsets.
- This is the most complete global set of historical tropical cyclones available. It includes data from 12 different agencies or historical databases

https://climatedataguide.ucar.edu/climate-data/ibtracs-tropical-cyclone-best-track-data

A use case would be to factor in this kind of covariate into PA impact evaluations (like droughts, rainfalls, or fires). I will expand this description later. Reactions from colleagues are welcome.

**Comment by Jo-Schie** [comment]
I have already developed a quite long and well thought routine to process IbTracs dataset and I was considering adding it to the package at some point in time. I can add more information in a couple of days as soon as I get back to my private laptop.

**Comment by Jo-Schie** [comment]
Ah. Already found a script see [here](https://github.com/openkfw/mapme.protectedareas/blob/main/code/processing-scripts/02_hurricanes.R). If I recall well the routine was based on the idea of converting the line data from ibtracs to polygons (areas) which are likely affected by a hurricane depending on the wind strength. 

Not sure though if everything is there or if there was still another script.

**Comment by fBedecarrats** [comment]
> Ah. Already found the scripts See [here](https://github.com/openkfw/mapme.protectedareas/blob/main/code/development/hurricanes.R) and [here](https://github.com/openkfw/mapme.protectedareas/blob/main/code/processing-scripts/02_hurricanes.R). If I recall well the routine was based on the idea of converting the line data from ibtracs to polygons (areas) which are likely affected by a hurricane depending on the wind strength.
> 
> Maybe try this out and check the results. For me it looks quite good (subjectively judged).

Hi @Jo-Schie , the links are dead. Could you please share again these snippets? I will work again on this soon.

**Comment by Jo-Schie** [comment]
Yes. that repo was put on private again. I added now a new branch to mapme.indicators. Maybe you want to try out to add the ressource there? 

The scripts are in this [folder](https://github.com/mapme-initiative/mapme.indicators/tree/e7904827a06d9854af5df5f1dcd7bb021a791acc/R). 

If you do not pretend to add it to the package just download the files and I can delete the branch again. 

pinging @goergen95 fyi.
